File: fenyofa/fenyofa.py
bemenet = input().split()
fenyofa_db, hatar_ar = int(bemenet[0]), int(bemenet[1])

draga_fenyofa_arak = {"ar": [], "index": []}


for i in range(fenyofa_db):
    arak = int(input())
    # print(arak)

    if arak > hatar_ar:
        draga_fenyofa_arak["ar"].append(arak)
        draga_fenyofa_arak["index"].append(i)

# print(draga_fenyofa_arak)

if draga_fenyofa_arak:
    min_ar = min(draga_fenyofa_arak["ar"])
    hanyadik = draga_fenyofa_arak["ar"].index(min_ar)
    # print(hanyadik)
    index = draga_fenyofa_arak["index"][hanyadik]
    print(index+1, min_ar)
else:
    print(-1)

File: fenyofa/fenyofa.js
let input = [6, 5000, 2500, 5300, 1900, 2400, 8800, 5300];

let min = Infinity;
let index;

function fenyo(arr) {
  for (let i = 2; i < arr.length; i++) {
    if (arr[i] > arr[1] && arr[i] < min) {
      min = arr[i];
      index = i - 1;
    }
  }
  if (min === Infinity) {
    console.log(-1);
  } else {
    console.log(`${index} ${min}`);
  }
}

fenyo(input);
